playbooks/capsule/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- name: 'Set SELinux to Permissive'
+  selinux: policy=targeted state=permissive
+
+- name: 'Get Server Hostname'
+  shell: hostname -f
+  delegate_to: "{{ server }}"
+  register: server_fqdn
+- name: 'Get Server IP Address'
+  shell: ifconfig eth0 2>/dev/null|awk '/inet/ {print $2}'|sed 's/addr://'
```

Comment:
I believe I tried this and it didn't work on both EL6 and El7 and ended up settling on this format which isn't ideal. There may be a way to delegate grabbing the IP address via a fact from the server host that I had just not stumbled across yet.